src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrise = 40;
+  const discontSevenDay = 50;
+  const discontThreeDay = 20;
+
+  if (days >= 7) {
+    return basePrise * days - discontSevenDay;
+  }
+
+  if (days === 6) {
+    return basePrise * days - discontThreeDay;
+  }
+
+  if (days >= 3) {
+    return basePrise * days - discontThreeDay;
+  }
+
+  if (days < 3) {
```

Comment:
The check `days < 3` is redundant since all previous conditions cover cases when `days` is 3 or more. If none of the previous conditions are met, you can safely return `basePrice * days` without this check.